mtest/libmscore/albums/smallPianoAlbumTest.msca

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<museScore version="3.01">
+  <Album>
+    <name>My Album</name>
+    <Score>
+      <alias></alias>
+      <path>/home/boop/Documents/GSoC_2020/mscore/MuseScore/mtest/mscore/albums/Piano1.mscx</path>
```

Comment:
will `/home/boop` work when this is being run by Travis? I'd guess no.

Reply:
You are right, fixing the paths of the tests is on my ToDo list :-)

Reply:
Would be good to avoid absolute paths where possible to avoid leaking information about the user.

Reply:
Especially, in this case, absolute paths would need a very smart workaround to not fail. I was thinking to give the users the ability to select whether they want both relative and absolute paths when saving/exporting.